ci-isolation/src/main/java/com/exasol/adapter/document/files/ciisolation/CiIsolationApp.java

```diff
@@ -0,0 +1,22 @@
+package com.exasol.adapter.document.files.ciisolation;
+
+import com.exasol.ciisolation.aws.PolicyReader;
+import com.exasol.ciisolation.aws.ciuser.CiUserStack;
+
+import software.amazon.awscdk.core.App;
+
+/**
+ * This class defines a CloudFormation stack that creates a user for the CI of this project.
+ */
+public class CiIsolationApp {
+    public static void main(final String[] args) {
+        final App app = new App();
+        final PolicyReader policyReader = new PolicyReader();
+        new CiUserStack(app,
+                CiUserStack.CiUserStackProps.builder().projectName("abs-files-vs")
```

Comment:
did you change the name here? It must be different from the other project.

Reply:
changed it